patroni/validator.py

```diff
@@ -379,21 +396,37 @@ def __init__(self, *args: Any) -> None:
         self.args = args
 
 
-class Optional(object):
-    """Mark a configuration option as optional.
+class OneOf(object):
```

Comment:
maybe `AtMostOne`? (no idea why I thought it fits `*available_dcs`. I guess, "less than one" should not pass there)